google/cloud/bigtable/data/execute_query/_parameters_formatting.py

```diff
@@ -109,6 +110,16 @@ def _detect_type(value: ExecuteQueryValueType) -> SqlType.Type:
             "Cannot infer type of None, please provide the type manually."
         )
 
+    if isinstance(value, list):
+        raise ParameterTypeInferenceFailed(
```

Comment:
Can you declare this in the [execute_query docstring](https://github.com/googleapis/python-bigtable/blob/df08dcb553d354c782ebdf036f82bc1a77daf7c1/google/cloud/bigtable/data/_async/client.py#L567)? It looks like that's the public entrypoint for this

Reply:
Done